src/absil/illib.fs

```diff
@@ -20,9 +25,10 @@ let (>>>&) (x:int32) (n:int32) = int32 (uint32 x >>> n)
 let notlazy v = Lazy<_>.CreateFromValue v
 
 let inline isNonNull x = not (isNull x)
-let inline nonNull msg x = if isNull x then failwith ("null: " ^ msg) else x
+let inline nonNull msg x = if isNull x then failwith ("null: " + msg) else x
 let (===) x y = LanguagePrimitives.PhysicalEquality x y
 
+#if !FABLE_COMPILER
```

Comment:
That's fine, if that's your preference I'll try to replace all #if with feature conditionals.